kylin/src/main/java/org/apache/zeppelin/kylin/KylinInterpreter.java

```diff
@@ -172,7 +172,7 @@ private InterpreterResult executeQuery(String sql) throws IOException {
     return rett;
   }
 
-  private String formatResult(String msg) {
+  public String formatResult(String msg) {
```

Comment:
LGTM except it. How about changing the package of test class as same as this class and make this method as package-local?